### DISCUSSION TITLE
There was no view ID specified. You must provide a valid view ID to execute queries on Google Analytics.

### DISCUSSION THREAD

**Discussion by chotosite** [freekmurze/laravel-analytics#397] [Q&A]
Hi,

I have newly installed laravel-analytics and configured according to documentation but I am still facing the problem.

I have already checked the [issue ](https://github.com/spatie/laravel-analytics/issues/281) but found no solution.

I read the [comment](https://github.com/spatie/laravel-analytics/issues/76#issuecomment-228536154) but did not understand how to check output of `config('laravel-analytics');` 

Any help will be very appreciated. Thank you..

**Comment by yergunes** [comment]
I think the problem is about the new Google Analytics Version (GA4). The new version doesn't include viewID so we can't add it to our config. But if as far as I can found we can install the older (Universal) version of analytics. So with that version, you can make it work. ([https://support.google.com/analytics/answer/10269537?hl](https://support.google.com/analytics/answer/10269537?hl))

**Reply by bartclaeys** [reply]
So are you saying this package does not work with GA4?

**Comment by sanjaydeveloper15** [comment]
Hi 👋

I am getting login required error

I am using json file and view ID both
As per documentation there is no option given where we need to put oauth access token

So do you have any idea regarding it ?

**Reply by freekmurze** [reply]
Please stop spamming our discussion. Asking your question once is enough 👍
